### PR TITLE
debts for yourself

### PR DESCRIPTION
**Pull request by AntonLazovsky**

### CODE REVIEW

File: imports/ui/components/debts/debts.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 Template.debts.helpers({
     debtsOrders: () => {
         let orders = [];
+        const userPoolIds = Pools.find({ ownerId: Meteor.userId() }, { _id: 1 })
```
Comment (denisbalyko) [inline imports/ui/components/debts/debts.js]: { _id: 1 }   ??

File: imports/ui/components/pool_summary/pool_summary.js [hunk 2]
```diff
@@ -12,6 +12,9 @@ Template.poolSummary.events({
         const pool = Pools.findOne({ _id: Template.instance().data._id });
         Pools.changePoolState(pool._id, utils.POOL_STATE.ARCHIVED);
 
+        let ownerOrder = Orders.findOne({ poolId: pool._id, userId: Meteor.userId() });
```
Comment (denisbalyko) [inline imports/ui/components/pool_summary/pool_summary.js]: const

File: imports/ui/components/common/common.js [hunk 3]
```diff
@@ -21,6 +21,10 @@ Template.registerHelper('formatTime', (time, format) => {
     return moment(time).format(format);
 });
 
+Template.registerHelper('isOrderOwner', (orderId) => {
+    return Meteor.userId() === Orders.findOne({ _id:orderId }).userId;
```
Comment (gaykov) [inline imports/ui/components/common/common.js]: пробел

File: imports/ui/components/common/common.js [hunk 4]
```diff
@@ -21,6 +21,10 @@ Template.registerHelper('formatTime', (time, format) => {
...
+});
+
 Template.registerHelper('isPoolOwner', (poolId) => {
```
Comment (gaykov) [inline imports/ui/components/common/common.js]: пробел

File: imports/ui/components/debts/debts.js [hunk 5]
```diff
@@ -1,8 +1,10 @@
...
+            .fetch().map(function(obj) { return obj._id; });
```
Comment (gaykov) [inline imports/ui/components/debts/debts.js]: function

**Pull request merged**